src/main/java/no/ndla/taxonomy/rest/v1/Subjects.java

```diff
@@ -183,8 +185,8 @@ public List<EntityWithPathChildDTO> getChildren(@PathVariable("id") URI id,
         return topicTreeSorter.sortList(filtered).stream().distinct().collect(Collectors.toList());
     }
 
-    private EntityWithPathChildDTO createChildDTO(Node subject, NodeConnection connection, String language) {
-        return new NodeChildDTO(subject, connection, language);
+    private NodeChildDTO createChildDTO(Node subject, NodeConnection connection, String language) {
+        return new NodeChildDTO(connection, language);
```

Comment:
Er det meningen at `subject` ikke skal være i bruk her?

Reply:
Det blei sånn når eg skreiv om. Node som blei sendt inn til EntityWithPathChildDTO blei uansett ikkje brukt til noe. Og sånn har det vore ganske lenge. Så funksjonalitet er uendra etter pr.

Reply:
Det ser ut som om den blei brukt til å plukke korrekt path for child-objekt, men dette er endra til å bruke parent i connection sin path. Som uansett er meir korrekt (trur eg).

Reply:
Ait nice!